### PR TITLE
staging to master!

### PR DESCRIPTION
**Pull request by ag-edge**
## Summary

Merging staging into master. New screenshots, content, and more. 

Link to Shortcut (issue tracker) story if available.

## Motivation

To match our current version off the product. 

## Checks

- [x] Run `npm run test` locally.
- [x] Run `npm run -s lint` locally.
- [x] Create a Pull Request (follow the [GitHub flow](https://guides.github.com/introduction/flow/)) and [Conventional Commits Specification](https://www.conventionalcommits.org/en/v1.0.0/))
- [x] Verify any structural/CSS changes on all screen sizes (if relevant)

### CODE REVIEW

**Review by Sophie-edgeworx — DISMISSED** [review]
LGTM

**Review by eplewis89 — DISMISSED** [review]
Everything LGTM

File: config/_default/params.toml [hunk 1]
```diff
@@ -54,10 +54,10 @@ lqipWidth = "20x"
 smallLimit = "300"
 
 # Footer
-footer = "©2023 Edgeworx Inc"
```
[the remark this answers is not in the document]
Comment (ag-edge) [reply config/_default/params.toml:57]: Makes sense! Do I need to add that and commit to get the desired outcome of a dynamic year?

**Review by eplewis89 — APPROVED** [review]

**Review by Sophie-edgeworx — APPROVED** [review]
LGTM

**Pull request merged**